app/assets/themes/fearlesscities/_variables.scss

```diff
@@ -0,0 +1,32 @@
+$fearless-yellow: #FFD21AFF;
+$fearless-blue: #554AE3FF;
+$fearless-red: #E43528FF;
+$fearless-black: #000000FF;
+
+$primary: $fearless-blue;
+$primary-rgb: 84, 74, 227;
+$secondary: $fearless-red;
+$secondary-rgb: 228, 53, 40;
+$success: $fearless-yellow;
+$success-rgb: 255, 210, 26;
+$warning: $fearless-red;
+$warning-rgb: 228, 53, 40;
+$alert: #f54433;
+$alert-rgb: 245, 68, 51;
+
+// Variables
+
+:root {
+  --primary: #{$primary};
+  --primary-rgb: #{$primary-rgb};
+  --secondary: #{$secondary};
+  --secondary-rgb: #{$secondary-rgb};
+  --success: #{$success};
+  --success-rgb: #{$success-rgb};
+  --warning: #{$warning};
+  --warning-rgb: #{$warning-rgb};
+  --alert: #{$alert};
+  --alert-rgb: #{$alert-rgb};
+}
+
```

Comment:
I suggest you overwrite the default `background-color` for different types of `callout` elements (otherwise it picks the `:root` variables for the border color and the "hardcoded" ones for the background)

Example from [Platoniq/decidim-pemb](https://github.com/Platoniq/decidim-pemb/blob/main/app/assets/stylesheets/pemb/partials/_variables.scss)
```scss
.callout.primary {
  border-left: $border-width-bigger solid $primary;
  background-color: hsl(map-get($hue, primary), 60, 90);
}

.callout.secondary {
  border-left: $border-width-bigger solid $bd-light-blue;
  background-color: hsl(map-get($hue, bd-light-blue), 60, 90);
}

.callout.success {
  border-left: $border-width-bigger solid $success;
  background-color: hsl(map-get($hue, success), 60, 90);
}

.callout.warning {
  border-left: $border-width-bigger solid $warning;
  background-color: hsl(map-get($hue, warning), 60, 90);
}

.callout.alert {
  border-left: $border-width-bigger solid $alert;
  background-color: hsl(map-get($hue, alert), 60, 90);
}
```